public/views/landing.html

```diff
@@ -13,7 +13,7 @@ <h1 style="text-align: center">CrowdData lets you access the knowledge of everyo
           <label class="control-label" for="password-input">Password</label>
           <input type="password" name="password" id="password-input" class="form-control"/>
         </div>
-        <button type="submit" class="btn btn-raised btn-lg btn-primary btn-block">Login</button>
+        <button type="submit" class="btn btn-raised btn-lg btn-default btn-block">Login</button>
```

Comment:
In the signup, the affirmative (signup) button is colored default and the negative (cancel) button is colored primary, so I updated this to reflect that coloring